### DISCUSSION TITLE
Provider field cannot be viewed in admin

### DISCUSSION THREAD

**Discussion by negati-ve** [strapi/strapi#5903] [Legacy Topics]
I have facebook provider enabled on Strapi beta.20 running on docker strapi/strapi image with mysql database

**Describe the bug**
I'm not sure if this is expected behavior 
I'm trying to see the provider of a user via strapi admin
but the field is not visible
if I try to Configure the view - User, it doesn't allow me to add provider as a field

**Steps to reproduce the behavior**

1. Register a user via external provider eg: facebook 
2. Go to collections 'User'
3. Click on any user

**Expected behavior**
Should be able to see provider in strapi admin under User collection  for each user

**Screenshots**
<img width="1413" alt="image" src="https://user-images.githubusercontent.com/13253073/79908558-1182ab80-8439-11ea-815e-fdbf47859ac2.png">
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/13253073/79908581-2101f480-8439-11ea-9076-d568ffd2c0b2.png">

**System**

- Node.js version: v12
- NPM version: 6.14.3
- Strapi version: 3.0.0-beta.20
- Database: Mysql
- Operating system: MacOS/Docker

**Comment by derrickmehaffy** [comment]
This is intended behavior as the provider is only used by the backend, is there a specific reason you want to view this? (you shouldn't be editing it manually)

**Comment by negati-ve** [comment]
2 things that are unclear and that maybe updated in the documentation 

1. Sometimes its handy to view which provider the user registered through.. even if we can't edit it, an option to view provider on strapi admin would be very useful

2. There's no clarity in the documentation on whether a user registered via local provider, when later logs in via an external provider has 2 accounts or if there is a mechanism to have new provider mapped to the original user

**Comment by lauriejim** [comment]
Hello! For the first question, @Aurelsicoko is the best person to answer this question.
For the second, this feature doesn't exist right now.
But you can code this logic by yourself by using the customization concept.
https://strapi.io/documentation/3.0.0-beta.x/concepts/customization.html
It's not the first time I see this question, I planned to create a [Jim's recipes](https://www.youtube.com/playlist?list=PL7Q0DQYATmvhlHxHqfKHsr-zFls2mIVTi) video on this topic.
You will have to update this function - https://github.com/strapi/strapi/blob/f56158b684454c90066ee3ee7ade0e6122a7f778/packages/strapi-plugin-users-permissions/controllers/Auth.js#L21 
That manages the login function.

**Comment by negati-ve** [comment]
Thanks @lauriejim for pointing in the right direction!

That clears up my doubts

Cheers!
